bedrock/settings/__init__.py

```diff
@@ -123,6 +143,48 @@
         FLUENT_REPO_PATH,
     ]
 
+    CANONICAL_LOCALES = {
+        # TODO: check whether redundant when we have real translations flowing in to Bedrock
+        "en-US": "en",
+        "en-GB": "en",
+        "en-CA": "en",
+        "es-ES": "es-ES",
+        "es-la": "es",  # NB: es-LA isn't a real locale, but it is what getpocket.com has used and we need to deal with it
+    }
+
+    FALLBACK_LOCALES = {
+        # TODO: double-check for correctness when we have real translations flowing in to Bedrock
+        # "es" is the fallback for non-Spain Spanish, not for "es-ES"
+        "es-CL": "es",
+        "es-MX": "es",
+        "es-AR": "es",
+    }
```

Comment:
As mentioned in Slack, we can drop these if the country specific locale maps to the 2 letter locale, since that's the default in our middleware for all locales. We only need entries here if we are mapping country specific to another country specific locale.